src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -124,7 +124,7 @@ Learn more in our [Fragment documentation](/platforms/android/configuration/inte
 
 ### App Start Instrumentation
 
-The App Start Instrumentation provides insight into how long your application takes to launch. It adds a span from the application launch to the first auto-generated UI transaction.
+The App Start Instrumentation provides insight into how long your application takes to launch. It adds a span from the application launch to the first auto-generated UI transaction. Also, the SDK sets the Span operation to `app.start.cold` or `app.start.warm`.
```

Comment:
`m`: I'm curious on why you put that sentence here? The docs mention cold and warm start directly below. This seems to be a bit redundant information to me.

Reply:
every auto instrumentation should mention the operation names, so people can look at it and filter, if here is the right place, maybe you are right, we could mention somewhere else

Reply:
moved to a new line, thanks